src/components/TopBar.js

```diff
@@ -208,7 +209,7 @@ export default _.flow(
             },
             href: Nav.getLink('root'),
             onClick: () => this.hideNav()
-          }, [logoGlow(), betaTag])
+          }, [menuOpenLogo, betaTag])
```

Comment:
The logo moves down and becomes larger pretty noticeably when I open the left-nav. 
Looking a little closer, it makes sense because the long logo has the cloud in-line next to the word FireCloud, and the open menu has the cloud centered. That being said, it still looks kinda weird to have the icon "jump" like that...not really sure what the best solution is tbh, but thought I'd call it out for further thoughts.